src/main/webapp/app/entities/study/study-dialog.component.ts

```diff
@@ -268,13 +291,18 @@ export class StudyDialogComponent implements OnInit {
         });
     }
 
+    // Used to load participants table in modal
+    private loadParticipants() {
+        document.getElementById("emailHeader").click();
```

Comment:
Using explicit element clicks really shouldn't be here — if this loads the participants then surely there's an angular way of initialising the view when they click on the tab?

Reply:
The closest I found was ngAfterViewInit(), and even then that's after the entire view is loaded, not an active tab.

I created a feature request for this next step listed at #138, so for now I think it's fine to leave it as is.